src/Admin/CitizenProjectAdmin.php

```diff
@@ -425,4 +428,25 @@ protected function configureListFields(ListMapper $listMapper)
             ])
         ;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getExportFields()
+    {
+        return [
+            'Nom' => 'name',
+            'Date de création' => 'createdAt',
+            'Catégorie' => 'category',
+            'Créateur' => 'createdBy',
+            'Membres' => 'membersCounts',
+            'Ville' => 'postAddress.cityName',
+            'Compétences recherchées' => 'skills',
```

Comment:
should you use `Comp.` to harmonize labels ?

Reply:
After discussing with @guillaumedenhez , we will keep "Compétences" for both.